Makefile

```diff
@@ -46,4 +48,5 @@ test-coverage:
 	$(foreach pkg, $(ALL_PACKAGES),\
 	go test -coverprofile=coverage.out -covermode=count $(pkg);\
 	tail -n +2 coverage.out >> coverage-all.out;)
-	@go tool cover -html=coverage-all.out -o coverage.html
\ No newline at end of file
+	@go tool cover -html=coverage-all.out -o coverage.html
+
```

Comment:
Shall we add a command to make a release using GoReleaser?

Reply:
As of now, everytime we push a version tag to master, it automatically makes a release. What you think?

Reply:
Yeah, how about a command `make release v.3.0.0` which create a Git tag with an appropriate tag message?

Reply:
`make release VERSION=v3.0.0` will add a git tag and can trigger the dunner release.

Reply:
Aweomse, now this looks complete! Good job!